internal/tss/channel.go

```diff
@@ -0,0 +1,34 @@
+package tss
+
+type RPCGetIngestTxResponse struct {
+	// a status that indicated whether this transaction failed or successly made it to the ledger
+	Status string
+	// the raw TransactionEnvelope XDR for this transaction
+	EnvelopeXdr string
```

Comment:
I think we should try to stick to [Go's styleguide](https://google.github.io/styleguide/go/decisions#initialisms) as much as possible
```suggestion
	EnvelopeXDR string
```

Reply:
Done